apps/web/src/pages/settings/tabs/EmailSettings.tsx

```diff
@@ -0,0 +1,175 @@
+import { useClipboard } from '@mantine/hooks';
+import styled from '@emotion/styled';
+import { inputStyles } from '../../../design-system/config/inputs.styles';
+import Card from '../../../components/layout/components/Card';
+import { ActionIcon, Center, Input as MantineInput } from '@mantine/core';
+import { colors, Text, Input, Tooltip, Button } from '../../../design-system';
+import { Check, SuccessIcon, Copy } from '../../../design-system/icons';
+import React, { useEffect } from 'react';
+import { Control, Controller, useForm } from 'react-hook-form';
+import { useEnvController } from '../../../store/use-env-controller';
+import { useMutation } from '@tanstack/react-query';
+import { updateDnsSettings, validateMxRecord } from '../../../api/environment';
+import { showNotification } from '@mantine/notifications';
+import { WarningIcon } from '../../../design-system/icons/statuses/WarningIcon';
+
+export const EmailSettings = () => {
+  const clipboardEnvironmentIdentifier = useClipboard({ timeout: 1000 });
+  const MAIL_SERVER_DOMAIN = '10 postal-dev.novu.co';
```

Comment:
Can this be configured in an .env file and fall back to us?

Reply:
Done